shell/browser/net/proxying_url_loader_factory.cc

```diff
@@ -538,8 +541,7 @@ void ProxyingURLLoaderFactory::InProgressRequest::ContinueToBeforeRedirect(
 
   factory_->web_request_api()->OnBeforeRedirect(&info_.value(), request_,
                                                 redirect_info.new_url);
-  target_client_->OnReceiveRedirect(redirect_info,
-                                    std::move(current_response_));
+  target_client_->OnReceiveRedirect(redirect_info, current_response_.Clone());
```

Comment:
Should we also clone the response in `target_client_->OnReceiveResponse` ?

Reply:
Yeah it should also be changed, thanks for noticing this.